edxval/urls.py

```diff
@@ -28,4 +28,9 @@
         views.get_subtitle,
         name="subtitle-content"
     ),
+    url(
+        r'^videos/update_video_images/$',
```

Comment:
would be more readable if:
`r'^videos/video-images/update/$',`
or
`r'^videos/generated-video-images/update/$',`